test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -65,11 +65,17 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 
 	async publish( { visit = false, closePanel = true } = {} ) {
 		await driverHelper.clickWhenClickable( this.driver, this.prePublishButtonSelector );
+
+		// Clicking publish too soon often fails, I suspect this is due to the slide out animation.
+		// A short sleep is enough to prevent these failures
+		await this.driver.sleep( 10 );
 		await driverHelper.clickWhenClickable( this.driver, this.publishButtonSelector );
-		// Let's give publishing request enough time to finish. Sometimes it takes
-		// way more than the default 20 seconds, and the cost of waiting a bit
-		// longer is definitely lower than the cost of repeating the whole spec.
-		await driverHelper.waitTillNotPresent( this.driver, this.publishingSpinnerSelector, 60000 );
```

Comment:
The 60 seconds timeout [has been recently added](https://github.com/Automattic/wp-calypso/pull/51575) to address issues where the publish callback was taking longer than usual causing unnecessary retries. Let's bring it back.

Reply:
I'm a little concerned about this one, how often does this happen? don't we want to know about it? 60 seconds sounds like a severe problem with the publish api.

Reply:
fyi #51863 replaced this line anyway